unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL.Emotes;
+using DCL.EmotesCustomization;
+using DCL.Tasks;
+using System;
+using System.Linq;
+using System.Threading;
+using UnityEngine;
+
+namespace DCL.Backpack
+{
+    public class BackpackEmotesSectionController : IBackpackEmotesSectionController
+    {
+        public event Action<string> OnNewEmoteAdded;
+        public event Action<string> OnEmotePreviewed;
+        public event Action<string> OnEmoteEquipped;
+        public event Action<string> OnEmoteUnequipped;
+
+        private DataStore dataStore;
+        private IUserProfileBridge userProfileBridge;
+        private IEmotesCatalogService emotesCatalogService;
+        private IEmotesCustomizationComponentController emotesCustomizationComponentController;
+        private CancellationTokenSource loadEmotesCts = new ();
+
+        public void Initialize(
```

Comment:
why the initialize method? cant you inject the dependencies?

Reply:
Yep! Done: https://github.com/decentraland/unity-renderer/pull/4933/commits/1bd714c8d9683981056f6e52f3e3ba20f6dd0ca0